images/build/Dockerfiles/cliBuilder.Dockerfile

```diff
@@ -0,0 +1,157 @@
+ARG DEBIAN_FLAVOR
+
+# Use the curl flavor of buildpack-deps as the base image, which is lighter than the standard flavor; more information here: https://hub.docker.com/_/buildpack-deps
+FROM buildpack-deps:${DEBIAN_FLAVOR}-curl as main
+ARG DEBIAN_FLAVOR
+ARG SDK_STORAGE_BASE_URL_VALUE="https://oryx-cdn.microsoft.io"
+ARG AI_CONNECTION_STRING
+ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
+
+COPY --from=oryxdevmcr.azurecr.io/private/oryx/buildscriptgenerator /opt/buildscriptgen/ /opt/buildscriptgen/
+COPY --from=oryxdevmcr.azurecr.io/private/oryx/support-files-image-for-build /tmp/oryx/ /opt/tmp
+
+ENV ORYX_SDK_STORAGE_BASE_URL=${SDK_STORAGE_BASE_URL_VALUE} \
+    ENABLE_DYNAMIC_INSTALL="true" \
+    PATH="/usr/local/go/bin:/opt/python/latest/bin:/opt/oryx:/opt/yarn/stable/bin:/opt/hugo/lts:$PATH" \
+    DYNAMIC_INSTALL_ROOT_DIR="/opt" \
+    PYTHONIOENCODING="UTF-8" \
+    LANG="C.UTF-8" \
+    LANGUAGE="C.UTF-8" \
+    LC_ALL="C.UTF-8" \
+    ORYX_AI_CONNECTION_STRING="${AI_CONNECTION_STRING}" \
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE="1"
+
+# Install an assortment of traditional tooling (unicode, SSL, HTTP, etc.)
+# Also install some PHP requirements (unsure why it's gated by buster flavor, maybe due to some libraries being released per debian flavor, but we should try to install these pre-reqs no matter what)
+RUN if [ "${DEBIAN_FLAVOR}" = "buster" ]; then \
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
+            libicu63 \
+            libcurl4 \
+            libssl1.1 \
+            # PHP pre-reqs
+            ca-certificates \
+            libargon2-0 \
+            libcurl4-openssl-dev \
+            libedit-dev \
+            libonig-dev \
+            libncurses6 \
+            libsodium-dev \
+            libsqlite3-dev \
+            libxml2-dev \
+            xz-utils \
+            # ruby pre-req
+            libyaml-dev \
+        && rm -rf /var/lib/apt/lists/* ; \
+    elif [ "${DEBIAN_FLAVOR}" = "bullseye" ]; then \ 
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
+            libicu67 \
+            libcurl4 \
+            libssl1.1 \
+            libyaml-dev \
+            libxml2 \
+        && rm -rf /var/lib/apt/lists/* ; \
+    else \
+        apt-get update \
+        && apt-get install -y --no-install-recommends \
+            libcurl3 \
+            libicu57 \
+            liblttng-ust0 \
+            libssl1.0.2 \
+        && rm -rf /var/lib/apt/lists/* ; \
+    fi
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends \
+        # .NET Core dependencies for running Oryx
+        libc6 \
+        libgcc1 \
+        libgssapi-krb5-2 \
+        libstdc++6 \
+        zlib1g \
+        # For .NET Core 1.1 (I believe we can remove this, .NET Core 1.1 has been 'End of Support" for three years)
+        libuuid1 \
+        libunwind8 \
+        rsync \
+        libgdiplus \
```

Comment:
Honestly, I've got no idea. @cormacpayne any thoughts?

Reply:
I assume this comment is about the .NET Core 1.1 comment, which applies to the two libraries below it (according to [this original commit](https://github.com/microsoft/Oryx/commit/03e7f76590d343c8d34907519b47b3f3bee8c405)) -- I would like to think we're OK to remove them since we don't support .NET Core 1.1 anymore, but those two packages are installed as a part of other build Dockerfiles (such as GitHub Actions), so it may be worth taking them out of those Dockerfiles as well and seeing if any tests fail? 🤔

Reply:
Ok, I'll do some testing around with the removal of these. Thanks!

Reply:
Looks like we still need libunwind2 according to https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-2204#dependencies, but i'm trying to remove libuuid1

Reply:
Sounds good! Thanks for updating Paul!

Reply:
Had to move around some other packages (for some reason we were only installing PHP deps for buster), but it seems like this was safely removed. Thanks for bringing it up!